public/faq.php

```diff
@@ -0,0 +1,156 @@
+                </div>
+              </div>
+            </div>
+  
+            <div class="card">
+              <div class="card-header" id="headingFour">
+                <a class="" data-toggle="collapse" href="#collapseFour" aria-expanded="false" aria-controls="collapseFour">
+                  <h5 class="mb-0 faq-card-heading">
+                    How secured are my savings?
+                    <span class="faq-getting-started-icon">+</span>
+                  </h5>
+                </a>
+              </div>
+              <div id="collapseFour" class="collapse" aria-labelledby="headingFour" data-parent="#accordion">
+                <div class="card-body">
+                  Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod.
+                </div>
+              </div>
+            </div>
+  
+            <div class="card">
+              <div class="card-header" id="headingFive">
+                <a class="" data-toggle="collapse" href="#collapseFive" aria-expanded="false" aria-controls="collapseFive">
+                  <h5 class="mb-0 faq-card-heading">
+                    Can I upgrade my saving plan in the future?
+                    <span class="faq-getting-started-icon">+</span>
+                  </h5>
+                </a>
+              </div>
+              <div id="collapseFive" class="collapse" aria-labelledby="headingFive" data-parent="#accordion">
+                <div class="card-body">
+                  Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod.
+                </div>
+              </div>
+            </div>
+  
+          </div>
+    </div>
+</section>
+
+<section class="faq-question">
+    <h2 class="faq-question-heading">You still have a question?</h2>
+    <h5 class="faq-question-text">If you cannot find answer to your question in our FAQ, you can <br> always contact us . We will be with you shortly!</h5>
+
+    <div class="faq-question-contact col-12">
+        <div class="col-lg-4 col-md-5 col-12 faq-question-info-div">
+            <img src="./assets/img/icons/phone-icon.png" alt="Phone icon">
+            <h4 class="faq-question-info">+234 807 654 5543</h4>
+            <p class="faq-question-info-small">We are always happy to help.</p>
+        </div>
+        &nbsp;&nbsp;&nbsp;
+        <div class="col-lg-4 col-md-5 col-12 faq-question-info-div">
+            <img src="./assets/img/icons/mail-icon.png" alt="Mail icon">
+            <h4 class="faq-question-info">support@ballers.ng</h4>
```

Comment:
This should be an anchor tag so that we can use the `mailto:` functionality in href to enable the visitor to send us a direct mail when clicked?